### PR TITLE
Added IMAGE attribute functionality

### PR DESCRIPTION
**Pull request by TheNewSound**
Feedback please! ;-)
This probably needs a new Property\Event\Image type?

### CODE REVIEW

**Comment by TheNewSound** [comment]
IMAGE is in [RFC 7986](https://tools.ietf.org/html/rfc7986#section-5.10)

**Comment by dereuromark** [comment]
Cool! Any plans of adapting this?

File: tests/Eluceo/iCal/Component/CalendarIntegrationTest.php [hunk 1]
```diff
@@ -141,4 +141,55 @@ public function testRenderIsIdempotent()
...
+     */
+    public function testEventAddImage()
+    {
+        $vCalendar = new \Eluceo\iCal\Component\Calendar('www.example.com');
+
+        $vEvent = new \Eluceo\iCal\Component\Event('123456');
+        $vEvent->setDtStart(new \DateTime('2012-12-24'));
+        $vEvent->setDtEnd(new \DateTime('2012-12-24'));
+        $vEvent->setSummary('Vacations');
+        $image = Array(
```
Comment (navarr) [inline tests/Eluceo/iCal/Component/CalendarIntegrationTest.php]: There should probably be a new image type

Comment (julienbourdeau) [reply tests/Eluceo/iCal/Component/CalendarIntegrationTest.php]: Yeah it would be easier to use with setters 👌

Comment (williamdes) [inline tests/Eluceo/iCal/Component/CalendarIntegrationTest.php]:
```suggestion
        $image = array(
```

**Pull request closed without merging**